up-squared.coffee

```diff
@@ -0,0 +1,55 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+UPBOARD_FLASH = '''
+	Power up the <%= TYPE_NAME %>. Make sure you have a keyboard connected.
+	Press the F7 key while BIOS is loading in order to enter the boot menu. Next, select the "UEFI:" option.
+'''
+
+UPBOARD_POWERRESET = 'Unplug and plug power on your <%= TYPE_NAME %>.'
+
+postProvisioningInstructions = [
+	instructions.BOARD_SHUTDOWN
+	instructions.REMOVE_INSTALL_MEDIA
+	UPBOARD_POWERRESET
+]
+
+module.exports =
+	version: 1
+	slug: 'up-squared'
+	name: 'UP Squared'
+	arch: 'amd64'
+	state: 'released'
+
+	stateInstructions:
+		postProvisioning: postProvisioningInstructions
+
+	instructions: [
+		instructions.ETCHER_USB
+		instructions.EJECT_USB
+		instructions.FLASHER_WARNING
+		UPBOARD_FLASH
+	].concat(postProvisioningInstructions)
+
+	gettingStartedLink:
+		windows: 'https://www.balena.io/docs/learn/getting-started/up-squared/nodejs/'
+		osx: 'https://www.balena.io/docs/learn/getting-started/up-squared/nodejs/'
+		linux: 'https://www.balena.io/docs/learn/getting-started/up-squared/nodejs/'
+
+	yocto:
+		machine: 'up-squared'
+		image: 'balena-image-flasher'
+		fstype: 'balenaos-img'
+		version: 'yocto-kirkstone'
+		deployArtifact: 'balena-image-flasher-up-squared.balenaos-img'
+		compressed: true
+
+	configuration:
+		config:
+			partition:
+				primary: 1
+			path: '/config.json'
+
+	options: [ networkOptions.group ]
+
+	initialization: commonImg.initialization
```

Comment:
The Up Board image we have already supports Up Squared too.
The only difference I am aware of are the ACPI tables that need to be used. And we allow to specify which ACPI table needs to be used via the dashboard

Reply:
Just out of curiosity, how would that work exactly?

Reply:
You can do that from the `Configuration` tab in dashboard:

```
Define SSDT overlays. Only supported by supervisor versions >= v. Only supported by supervisor versions >= v10.9.2.

```